src/main/webapp/index.html

```diff
@@ -60,17 +60,19 @@
             <input id="link-input" name="link-input" required pattern="https?://www.youtube.com/watch\?v=.+"/>
             <small id="url-help" class="form-text text-muted">Format: https://youtube.com/watch?v=...</small>
             </br>
-            <button type="submit" class="btn btn-primary submitBtn" id="submit-btn" onClick="updateUIWithVideoContext();">Enter</button>
+            <button type="submit" class="btn btn-primary" id="submit-link-btn">Enter</button>
             <br>
             </br>
           </div>
         </div>
-        <p>Maximum number of comments: </p>
+        <div id="comment-slider">
```

Comment:
In a separate PR could we hide the comment-slider after the user clicks the submit button?

Reply:
This could be cool!